bears/vimscript/VintBear.py

```diff
@@ -12,7 +13,7 @@ class VintBear:
     """
 
     LANGUAGES = {'VimScript'}
-    REQUIREMENTS = {PipRequirement('vim-vint', '0.3.10')}
+    REQUIREMENTS = {PackageRequirement('broken', 'vim-vint', '0.3.10')}
```

Comment:
Nothing.
It is just a value for the mandatory `type`, and this is a bad hack which can be easily backported.